src/utils/Normalize.scss

```diff
@@ -0,0 +1,8 @@
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+
+label,
```

Comment:
Consider using more specific selectors to avoid unintended side effects. For example, use `.form-label` instead of `label`. 🎯